src/common/src/system_param/reader.rs

```diff
@@ -31,67 +50,71 @@ impl From<PbSystemParams> for SystemParamsReader {
     }
 }
 
-impl SystemParamsReader {
-    pub fn barrier_interval_ms(&self) -> u32 {
+// Unwrap the field if it exists from the initial release.
+// Otherwise, fill it with the backward compatibility logic.
```

Comment:
What is the "release"? and what is compat logic?

Reply:
I kind of get it, but still confused. e.g., why can we unwarp `wasm_storage_url`

Reply:
It must be a bug. 😄

Reply:
... and why can we unwrap `state_store`, which defaults to `None`

Reply:
It's not related with the `Option` in prost at all.

Reply:
It's okay. 😄 It'll be checked on startup.

https://github.com/risingwavelabs/risingwave/blob/ffd922ff26ab4a374a08e09178b2a5e6f3f8e135/src/meta/src/manager/system_param/mod.rs#L59-L70